src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/notification/INotification.java

```diff
@@ -0,0 +1,52 @@
+/*
+ *  The MIT License
+ *
+ *  Copyright 2010 Sony Ericsson Mobile Communications. All rights reserved.
+ *  Copyright 2012 Sony Mobile Communications AB. All rights reserved.
```

Comment:
_nit_ Since you are the author of these new files you should technically add your own copyright (or your employer's depending on your contract).
I don't know if you can give away your copyright like this, but IANAL :shrug: In the git world the code is associated with your email anyway, so the original author can be found.

Reply:
I worked on these changes in my spare time. Adapted it for the new files, but I don't mind in general, happy to give something back to the community.